gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -137,7 +139,11 @@ fn v1_routes(gateway: Gateway) -> Router {
     // Public routes on gateway webserver
     let public_routes = Router::new()
         .route("/pay_invoice", post(pay_invoice))
-        .route("/id", get(get_gateway_id));
+        .route("/id", get(get_gateway_id))
+        // These routes are for next generation lightning
```

Comment:
I assume that means we don't provide any stability guarantees for these, so don't need to bump the API version.